docs/disciplines/Pharmacogenomics/Navigating_PSets.md

```diff
@@ -0,0 +1,69 @@
+# Navigating PSets
```

Comment:
Add - Why would anyone use a PSet. Hint - standardized normalization across datasets, names inside dataframes, identifiers for drugs, cell lines etc.